support/android/apk/app/build.gradle

```diff
@@ -0,0 +1,302 @@
+apply plugin: 'com.android.application'
+
+
+import groovy.io.FileType
+import org.apache.tools.ant.taskdefs.condition.Os
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+
+android {
+    compileSdkVersion 25
+    buildToolsVersion "25.0.2"
+
+    defaultConfig {
+        applicationId "com.mozilla.servo"
+        minSdkVersion 18
```

Comment:
I agree that setting target 21 would simplify parts of the code (is the minimum api level for native arm64 builds too). But Jelly bean + KitKat still has around 35-40% market share (https://developer.android.com/about/dashboards/index.html). What do you think about this @larsbergstrom ?